ecs-cli/modules/cli/compose/entity/service/service.go

```diff
@@ -392,6 +397,10 @@ func (s *Service) buildCreateServiceInput(serviceName, taskDefName string) (*ecs
 		createServiceInput.HealthCheckGracePeriodSeconds = aws.Int64(*s.healthCheckGP)
 	}
 
+	if len(s.serviceRegistries) > 0 {
```

Comment:
Could you remind me again where service registries is coming from? Flags?

Reply:
Service registries is the service discovery service that we created. I modeled this code after what was done for health check grace period- serviceRegistries is a field on the Service struct which will be set if Service Discovery is used. Then the `buildCreateServiceInput` function just checks if it exists or not to determine if gets included in the CreateService request.